### PR TITLE
[17.0][MIG] sales_team_security

### PR DESCRIPTION
**Pull request by Deriman-Alonso**
Standart migration.

### CODE REVIEW

**Review by christian-ramos-tecnativa — APPROVED** [review]
LGTM!

**Review by flotho — APPROVED** [review]
code review,
Thanks

File: sales_team_security/hooks.py [hunk 1]
```diff
@@ -33,7 +32,6 @@ def uninstall_hook(cr, registry):  # pragma: no cover
         )
     # At installation time, we need to sync followers
     with api.Environment.manage():
```
Comment (pedrobaeza) [inline sales_team_security/hooks.py]: No need of this.

Comment (Deriman-Alonso) [reply sales_team_security/hooks.py]: Your right, change done, Ty 👍.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**